electron-app/src/components/CreateAccount.vue

```diff
@@ -93,7 +93,10 @@ export default class CreateAccount extends Vue {
 
   readonly passwordRules = [(v: string) => !!v || 'Please provide a password'];
   readonly passwordConfirmRules = [
-    (v: string) => !!v || 'Please provide a password confirmation'
+    (v: string) => !!v || 'Please provide a password confirmation',
+    (v: string) =>
+      v == this.password ||
```

Comment:
@kelsos this does not work. It is always false. How can I properly compare the password confirm to the password field for validation purposes in the UI?

Also can there be a unit test for this? Sounds unit-testable.

Reply:
Yes, the problem is that the rule is getting passed the initial string reference in the beginning, and then when the password changes the reference changes. This means that for the rule `this.password` is always an empty string (a reference to the initial string). We could probably work around that by adding a computed property.

Reply:
What is a computed property? How does that work?